apps/design/frontend/src/contests_screen.tsx

```diff
@@ -391,6 +391,7 @@ function ContestForm({
                         <Input
                           type="text"
                           value={candidate.name}
+                          autoFocus
```

Comment:
Unrelated change to make it a bit quicker to type in candidate names in sequence